README.md

```diff
@@ -140,7 +140,11 @@ scoop install alacritty
 ### Other
 
 Prebuilt binaries for Linux, macOS, and Windows can be downloaded from the
-[GitHub releases page](https://github.com/jwilm/alacritty/releases).
+[GitHub releases page](https://github.com/jwilm/alacritty/releases). Note
+that some desktop environments have trouble rendering SVG icons. For these
```

Comment:
I guess it should be: `For those`.

Reply:
I think both would be fine here, but it might sound a bit strange because of the sentence structure regardless.

This might flow a bit nicer:

```
If your desktop environment has trouble rendering the default SVG icons, you can find a PNG and simplified SVG in the `extra/logo/compat` directory.
```